src/fdk_rdf_parser/classes/public_service.py

```diff
@@ -12,6 +12,7 @@
 @dataclass
 class PublicService(Service):
     hasCompetentAuthority: Optional[List[Organization]] = None
+    specializedType = "publicservices"
```

Comment:
for events er verdiene til specialized_type snake_case og i entall, kanskje greit å holde det likt?
```suggestion
    specializedType = "public_service"
```

Reply:
Eventuelt hvis du skal beholde det du bruker i fdk-portal:
```suggestion
    specializedType = "publicService"
```

Reply:
Datasettserier brukes `datasetSeries`, så vil uansett bli inkonsekvent.

Reply:
Men skriveleif ja, at det ikke ble camelcase, bra catch!